group.py

```diff
@@ -1,5 +1,83 @@
 """An example of how to represent a group of acquaintances in Python."""
 
 # Your code to go here...
+def max_age(group_information):
+    max_age = 0
+    people = None
+    for key, value in group_information.items():
+        if max_age < value['age']:
+            max_age = value['age']
+            people = key
+
+    print("The older people of this group is " + people + " and the age is " + str(max_age))
```

Comment:
This is a perfect place for f strings, which let you format strings, even running code inline
```suggestion
    print(f"The maximum age is {max(ages)}")
```